metrics-otlp/README.md

```diff
@@ -39,11 +40,11 @@ Furthermore it brings an OpenTelemetry Collector DaemonSet acting as agent runni
 
 ## Install the gateway
 
-1. Deploy the gateway
+1. Deploy the gateway.
 
-   Deploy an otel-collector using the upstream helm chart with prepared [values](./metrics-gateway-values.yaml). The values file is defining a pipeline for receiving OTLP metrics, is enriching them with resource attributes fullfilling the kubernetes semantic conventions, and then exports them to a custom OTLP backend.
+   Deploy an Otel Collector using the upstream Helm chart with a prepared [values](./metrics-gateway-values.yaml). The values file defines a pipeline for receiving OTLP metrics, enriches them with resource attributes that fulfil the Kubernetes semantic conventions, and then exports them to a custom OTLP backend.
    
-   As this instructions are not providing any backend, the following command sets the backend configuration with placeholders `myEndpoint` and `myToken` which needs to get adjusted to your needs. Be aware of, that a token should be provided using a Secret which gets mounted via the `extraEnvs` parameter. As this cannot be passed via the command itself in an easy way, please consider using a dedicated additional values.yaml file for that.
+   Because these instructions don't provide any backend, the following command sets the backend configuration with placeholders `myEndpoint` and `myToken`. Adjust them to your needs. It's recommended that you provide a token using a Secret, which is mounted with the `extraEnvs` parameter. Because the parameter  cannot be passed with the command easily, use a dedicated additional `values.yaml` file for that.
```

Comment:
Twice "because" is ugly, I didn't notice that in my first review. Also, I found a duplicate blank space :D 

```suggestion
   These instructions don't provide any backend, so use the following command to set the backend configuration with placeholders `myEndpoint` and `myToken`. Adjust them to your needs.
   It's recommended that you provide a token using a Secret, which is mounted with the `extraEnvs` parameter. Because the parameter cannot be passed with the command easily, use a dedicated additional `values.yaml` file for that.
```